### ISSUE TITLE
Error when attempting to append custom style tag to widget

### ISSUE THREAD

**Issue by rjacobson16** [Ajhad1/Zendesk-Customizations#1]
Inserted code: 

```
this.zE('webWidget:on', 'open', function() {
        const style = document.createElement('style');
        style.innerHTML = `
          /* hides the zendesk branding */
          /* inspect using developer tools to get the correct class */
          .ldASYI {
            font-family: 'Roboto' !important;
          }
          `;

          parent.document.getElementById('webWidget').contentDocument.head.appendChild(style);
        });
      </script>
  ```
  
Error:

```
Uncaught TypeError: Cannot read properties of null (reading 'contentDocument')
```

`parent.document.getElementById('webWidget')` returns null as well. Is your website not Cross Origin from your embedded widget?

**Comment by Ajhad1** [comment]
It is cross origin from the web widget. I am not sure why you are not able to find the web widget. My guess is that you are implementing your code before the web widget is initialized. This specific code needs to be implemented after the web widget initialization.

**Comment by noriktal** [comment]
Hello there,

I am having the same issue. I think that the chat iframe (#webWidget) is not yet inside the DOM when the callback of zE('webWidget:on', 'open', function() {}) fires.

**Comment by Ingirorhaun** [comment]
> Hello there,
> 
> I am having the same issue. I think that the chat iframe (#webWidget) is not yet inside the DOM when the callback of zE('webWidget:on', 'open', function() {}) fires.

I agree, I also experienced the same issue and solved it by wrapping `parent.document.getElementById('webWidget').contentDocument.head.appendChild(style);` in a `setTimeout` function
